src/frontend/komponenter/ManuellJournalføring/ManuellJournalføring.tsx

```diff
@@ -19,8 +19,21 @@ import HentPerson from '../Felleskomponenter/HentPerson/HentPerson';
 import UIModalWrapper from '../Felleskomponenter/Modal/UIModalWrapper';
 import Skjemasteg from '../Felleskomponenter/Skjemasteg/Skjemasteg';
 import { KnyttTilBehandling } from './KnyttTilBehandling';
+import { Dokumenter } from './Dokumenter';
+import styled from 'styled-components';
 import { useAmplitude } from '../../utils/amplitude';
 
+const PageSplit = styled.div`
+    display: flex;
+    flex-direction: row;
```

Comment:
I think the flex-direction is row by default. Do we need to set it here explicitly?  🤔

Reply:
No we dont need it:)